src/mongosh/schemas/curriculum-2.0.0.json

```diff
@@ -1,33 +1,49 @@
 {
-	"description": "A Learning resource or other activity that is on a persons career or learning roadmap",
+	"description": "A Learning resource or other activity that is on a person's career or learning roadmap",
 	"bsonType": "object",
 	"required": [
-		"_id"
+		"_id",
+		"personId",
+		"resources"
```

Comment:
I think ``personID`` is no longer needed as ``_id`` will be the ``perseonId``
Be careful about requiring resources - when we first create a curriculum it will not have anything other than ``_id``.

Reply:
okay got it.